webgpu.h

```diff
@@ -149,20 +149,20 @@ struct WGPURequestAdapterOptions;
 struct WGPUSamplerBindingLayout;
 struct WGPUSamplerDescriptor;
 struct WGPUShaderModuleCompilationHint;
-struct WGPUShaderModuleSPIRV;
-struct WGPUShaderModuleWGSL;
+struct WGPUShaderSourceSPIRV;
+struct WGPUShaderSourceWGSL;
 struct WGPUStencilFaceState;
 struct WGPUStorageTextureBindingLayout;
 struct WGPUSurfaceCapabilities;
 struct WGPUSurfaceConfiguration;
 struct WGPUSurfaceDescriptor;
-struct WGPUSurfaceFromAndroidNativeWindow;
-struct WGPUSurfaceFromCanvasHTMLSelector;
-struct WGPUSurfaceFromMetalLayer;
-struct WGPUSurfaceFromWaylandSurface;
-struct WGPUSurfaceFromWindowsHWND;
-struct WGPUSurfaceFromXcbWindow;
-struct WGPUSurfaceFromXlibWindow;
+struct WGPUSurfaceSourceAndroidNativeWindow;
+struct WGPUSurfaceSourceCanvasHTMLSelector_Emscripten;
```

Comment:
IIRC we actually want to remove this from this header (we'll make it an Emscripten-exclusive addon for now) but that doesn't need to block this PR.